### ISSUE TITLE
Filetype plugin disabled on Neovim

### ISSUE THREAD

**Issue by PreciousChicken** [PreciousChicken/vim-groff-viewer#9]
**Describe the issue**
Filetype plugin not recognised on Neovim (v0.9.1), i.e. none of the mappings work.  Continues to work on vim.  

**To Reproduce**
1. Open groff file using neovim
2. Enter `<leader>o`

Neovim will enter insert mode and not open viewer - as if plugin was not there.

**Expected behavior**
Viewer to open.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Manjaro 23.0.0
 - Neovim v0.9.1

**Additional context**
This is almost certainly an issue with the disabling option code:

```vimscript
if exists("b:did_ftplugin")
	finish
endif
let b:did_ftplugin = 1
```

If the variable name is changed then it will work in both vim and neovim.  Does neovim think a plugin for this filetype has already been activated (i.e. b:did_ftplugin= 1) and therefore this plugin is disabled?  However running `echo b:did_ftplugin` gives a value of 1 on both vim and neovim when running with `--clean` flag - why therefore is vim not disabling this plugin and neovim is?  Fix is to change the variable name, which does not seem to have any negative consequences, but unsure why neovim is now behaving this way.  May submit an issue with neovim - but need to determine exactly what problem is first.

**Comment by PreciousChicken** [comment]
Fixed in release [v2.0.2](https://github.com/PreciousChicken/vim-groff-viewer/releases/tag/v2.0.2).

**Comment by PreciousChicken** [comment]
Apparently now fixed in Neovim:

https://github.com/neovim/neovim/issues/24781#issuecomment-1683830915
